data/migrations/20191014160441_farmeruser.js

```diff
@@ -0,0 +1,24 @@
+
```

Comment:
Do you have an updated schema and relationships graph, I'm not understanding the use of this. It could be me though.

Reply:
The SQL to replace the farmers table would be:

`SELECT userID FROM users 
WHERE isfarmer="true";`

vs the old way

`SELECT userID FROM farmers`

Reply:
This is just both methods we talked about yesterday. We can simplify or remove things, but there's both a table for quick queries and a Boolean for checks on the frontend. Still working on this